### DISCUSSION TITLE
How do I delete items?

### DISCUSSION THREAD

**Discussion by tkurtbond** [200ok-ch/organice#773] [Q&A] [answered]
When I edit the title there is a trashcan on the right, but nothing seems to happen when I select it.

**Comment by schoettl** [comment]
Bug confirmed: The trashcan button has no effect. I created an issue.

For now, you can swipe a header to the left to delete.

**Comment by munen** [comment, marked as answer]
The trashcan deletes a TODO keyword if one is selected. The dots on the other side of the TODO keywords switch between TODO keyword sets if multiple ones are defined.

If you want to delete the headline, swipe the headline to the left.
